src/tokens/eth/0x050cbFF7BFf0432b6096DD15CD9499913dDF8E23.json

```diff
@@ -0,0 +1,35 @@
+{
+"symbol": "FCI",
+"name": "Futuregoldcoin",
+"type": "ERC20",
+"address": "0x050cbFF7BFf0432b6096DD15CD9499913dDF8E23",
+"ens_address": "",
+"decimals": 10,
```

Comment:
the decimals value seem to be incorrect
https://etherscan.io/token/0x050cbFF7BFf0432b6096DD15CD9499913dDF8E23